plugins/woocommerce/src/Blocks/BlockTypes/ProductFiltersActive.php

```diff
@@ -2,15 +2,22 @@
 namespace Automattic\WooCommerce\Blocks\BlockTypes;
 
 /**
- * CollectionAttributeFilter class.
+ * Product Filters: Active Block.
  */
-final class CollectionActiveFilters extends AbstractBlock {
+final class ProductFiltersActive extends AbstractBlock {
 	/**
 	 * Block name.
 	 *
 	 * @var string
 	 */
-	protected $block_name = 'collection-active-filters';
+	protected $block_name = 'product-filters-active';
+
+	/**
+	 * Interactivity namespace.
+	 *
+	 * @var string
+	 */
+	protected $interactivity_namespace = 'woocommerce/product-filters-active';
```

Comment:
Should we reuse the `$block_name` here or turn this into a method?

Reply:
Ah yeah, originally I didn't because I don't like `.` string concatenation, but a method is a good idea.

Reply:
Make sense! @kmanijak @imanish003 @thealexandrelara @roykho Product Collection and Product Gallery can use this I guess?